pkg/devfile/parser/data/2.0.0/components.go

```diff
@@ -27,12 +28,12 @@ func (d *Devfile200) GetCommands() []common.DevfileCommand {
 }
 
 // GetParent returns the  DevfileParent object parsed from devfile
-func (d *Devfile200) GetParent() common.DevfileParent {
+func (d *Devfile200) GetParent() apiComp.Parent {
```

Comment:
why apiComp?  how about `v1.Parent`?

Reply:
yes, v1 seems better, will switch to v1.

Reply:
resolved in 679c280